README.md

```diff
@@ -1,43 +1,42 @@
-# ESPuino - rfid-based musiccontroller based on ESP32 with I2S-DAC-support
+# ESPuino - RFID-controlled Audio Player based on ESP32 with I2S-DAC Support
+
+![build workflow](https://github.com/biologist79/ESPuino/actions/workflows/build.yml/badge.svg)
 
 ## Forum
 
 - EN: I've set up a primarily German-speaking community with much documentation. Also an
   international corner for non-German-speakers is available at https://forum.espuino.de.
-  Github-Login can be used there but it's not mandatory.
+  GitHub login can be used for signing in there (optional).
 - DE: Ich habe ein primär deutschsprachiges Forum aufgesetzt, welches ich mit reichlich Doku
   versehen habe. Würde mich freuen, euch dort zu sehen: https://forum.espuino.de. Ihr könnt euch
   dort mit eurem Github-Login einloggen, jedoch auch "normal" anmelden. Dokumentation findet ihr
-  insbesondere hier: https://forum.espuino.de/c/dokumentation/anleitungen/10
-
-## Build status
-
-![build workflow](https://github.com/biologist79/ESPuino/actions/workflows/build.yml/badge.svg)
+  insbesondere hier: https://forum.espuino.de/c/dokumentation/anleitungen/10.
 
 ## Current Development
 
-A [dev](https://github.com/biologist79/ESPuino/tree/dev)-branch was created. Any new features will
-be introduced and tested there first until they become part of the master-branch. Feel free to test
-but be advised this could be unstable.
+There is a [dev](https://github.com/biologist79/ESPuino/tree/dev)-branch that contains new features,
```

Comment:
I think it would be better to include '-branch' into the link. Makes it better visible and more easily clicked.
Maybe just write 'development branch'